src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -436,4 +436,25 @@
   <data name="SymWriterNotDeterministic" xml:space="preserve">
     <value>SymWriter doesn't support deterministic compilation</value>
   </data>
+  <data name="RuleSetBadAttributeValue" xml:space="preserve">
+    <value>The attribute {0} has an invalid value of {1}.</value>
+  </data>
+  <data name="RuleSetMissingAttribute" xml:space="preserve">
+    <value>The element {0} is missing an attribute named {1}.</value>
+  </data>
+  <data name="KeepAliveIsNotAnInteger" xml:space="preserve">
+    <value>Argument to '/keepalive' option is not a 32-bit integer.</value>
+  </data>
+  <data name="KeepAliveIsTooSmall" xml:space="preserve">
+    <value>Arguments to '/keepalive' option below -1 are invalid.</value>
+  </data>
+  <data name="KeepAliveWithoutShared" xml:space="preserve">
+    <value>'/keepalive' option is only valid with '/shared' option.</value>
+  </data>
+  <data name="MismatchedVersion" xml:space="preserve">
+    <value>Roslyn compiler server reports different protocol version than build task.</value>
```

Comment:
Is the use of the word "Roslyn" approved there? Maybe remove it entirely. "Compiler server reports ..."